### DISCUSSION TITLE
Storyboard does not work properly on this beatmap

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#20865] [Q&A] [answered]
This storyboard is black from 1:54 to 2:09
Storyboard seems to play early
This problem is not encountered in Stable
[Beatmap](https://osu.ppy.sh/beatmapsets/1503327#osu/3079907)

**Comment by Walavouchey** [comment, marked as answer]
part of this is an issue with the video playback, not the storyboard

the video is defined to start about 3 seconds before the audio, but it seems to somehow start even earlier / skip part of the video

```
Video,-3350,"video.mp4"
```

https://user-images.githubusercontent.com/36758269/197359720-6138db8c-f42d-4cf6-b0c1-7afc31b2eced.mp4

https://user-images.githubusercontent.com/36758269/197359723-a85227a7-e6f2-4ac8-a82e-76847ca47db8.mp4

---

the storyboard does define parts where it should be dimmed and black, but they don't line up with the video because of the above issue

however, the part where it's completely black for a while isn't intended

https://user-images.githubusercontent.com/36758269/197360004-c07a02e6-ad3b-4354-9985-74d4e970b8d4.mp4

The only weird thing i see in the storyboard is a backwards command for the sprite in question

```
Sprite,Overlay,Centre,"sb\black.jpg",320,240
 F,0,119041,119241,0,1
 F,0,120126,120626,1,0
 F,0,121210,121410,0,1
 F,0,122856,122756,1,0 // if timestamps and values are swapped, that would leave this black sprite at full opacity until 134944
                       // actual behaviour is to instantly fade to 0 at 122756
 S,0,134944,,16

```

seems like https://github.com/ppy/osu/pull/20683 was misinterpreting the issue and logic required.

**Reply by Walavouchey** [reply]
in fact, that pull request didn't even fix the [storyboard in the linked issue](https://osu.ppy.sh/beatmapsets/768768)

![screenshot002](https://user-images.githubusercontent.com/36758269/197361757-daf0520e-eed9-429b-9aa5-492d0f925e52.png)

![osu_2022-10-22_23-44-13](https://user-images.githubusercontent.com/36758269/197361759-3ddd5ccc-aafe-4f53-bf4a-460748bc82cd.jpg)

**Reply by smoogipoo** [reply]
Hope I've covered everything here...
Storyboard videos play too early: https://github.com/ppy/osu/issues/20898 (from this issue)
Reopened https://github.com/ppy/osu/issues/20632
